src/unlimited.cpp

```diff
@@ -1615,6 +1617,426 @@ UniValue setlog(const UniValue &params, bool fHelp)
 
     return ret;
 }
+
+/** Stratum Protocol - begin */
+
+
```

Comment:
Can we move these functions out of unlimited.cpp and create an /rpc/stratum.cpp or something similar?